config/kubernetes/sig-autoscaling/teams.yaml

```diff
@@ -38,53 +40,63 @@ teams:
     description: ""
     members:
     - gjtempleton
-    - maciekpytel
-    - mwielgus
+    - raywainman
```

Comment:
Why have a smaller subset of the leads here?

Reply:
+1, I missed this

Reply:
This was based on who had expressed interest in getting onto the HPA reviewers/approvers list.

Happy to extend to all of us if you both want to be in the loop as well.